crates/sui-framework/packages/move-stdlib/sources/type_name.move

```diff
@@ -47,6 +47,13 @@ public native fun with_defining_ids<T>(): TypeName;
 /// the package, even if the type in question was introduced in a later upgrade).
 public native fun with_original_ids<T>(): TypeName;
 
+/// Like `with_defining_ids`, this accesses the package ID that original defined the type `T`.
```

Comment:
both here and in the next function I would remove "Like `with_defining_ids`, this" but it's just me so whatever really

Reply:
I want some reference to those functions since they provide a bit more explanation and context. Though I am open to suggestions on how to fit that in there